src/newdb.cpp

```diff
@@ -1040,6 +1152,8 @@ bool load_char(const char *name, char_data *ch, bool logon, int pc_load_origin)
           obj->graffiti = str_dup(PFILES_INV_GRAFFITI);
 
         GET_OBJ_IDNUM(obj) = strtoul(PFILES_INV_OBJ_IDNUM, NULL, 0);
+        if (*PFILES_INV_MATRIX_RESTRING)
```

Comment:
`PFILES_INV_MATRIX_RESTRING && *PFILES_INV_MATRIX_RESTRING` since this can be null.